Unreal/SystemLocalized/det/Ceremony.det

```diff
@@ -12,11 +12,11 @@ LevelEnterText="Betrete den Zeremonienflügel"
 
 [TranslatorEvent15]
 ; EN: Message="The inner ceremonial chambers. Here the water god's serpents reside to accept the flesh of the weak to be devoured. Only a warrior who swims like the serpent shall escape their clutches."
-Message="Die inneren Zeremonienkammern. Hier leben die Schlangen des Wassergottes, die darauf heischen, das Fleisch der Schwachen zu verschlingen. Nur ein Krieger, der selbst wie die Schlange schwimmt, kann sich ihren Klauen entrinnen."
+Message="Die inneren Zeremonienkammern. Hier leben die Schlangen des Wassergottes, die darauf heischen, das Fleisch der Schwachen zu verschlingen. Nur ein Krieger, der selbst wie die Schlange schwimmt, kann ihren Krallen entrinnen."
```

Comment:
@SlyMk3 You mean "sich entziehen"? "Klauen" are something that hooved animals and birds have. "Krallen" seems like it would be the correct term for the Slith.

Reply:
"entrinnen" and "sich entziehen" also seem perfectly equal when it comes to an ancient stone tablet's phrasing.

Reply:
"...die darauf heischen" ???? What? Maybe ..."die darauf heiß sind"

Reply:
Changed it yet again:

"Die inneren Zeremonienkammern. Hier hausen die Schlangen des Wassergottes, welche die Schwachen ergreifen um ihr Fleisch zu verschlingen. Nur ein Krieger, der selbst wie die Schlange schwimmt, kann ihren Krallen entrinnen."